examples/canjs/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+!node_modules/canjs-localstorage/can.localstorage.js
+!node_modules/canjs/can.jquery.js
+!node_modules/jquery/dist/jquery.js
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-app-css/index.css
+!node_modules/todomvc-common/base.css
```

Comment:
look at line 1
They ignore node_modules/*

Which ignores all sub folders recursively, thus making any nested exclusions not work 
I know because I literally just just tested this and read the docs on the topic :(